cypress/features/regression/link.feature

```diff
@@ -22,31 +22,31 @@ Feature: Link component
 
   @positive
   Scenario: Link is tabbable
-    When I open default "Link Test" component in noIFrame with "link" json from "commonComponents" using "tabbable" object name
+    When I open default "Link Test" component with "link" json from "commonComponents" using "tabbable" object name
     Then Link is tabbable
 
   @positive
   Scenario: Link is not tabbable
-    When I open default "Link Test" component in noIFrame with "link" json from "commonComponents" using "tabbableFlase" object name
+    When I open default "Link Test" component with "link" json from "commonComponents" using "tabbableFlase" object name
```

Comment:
Should it be a spelling mistake for "tabbableFalse"?

Reply:
updated